### PR TITLE
merge to master PR

### PR DESCRIPTION
**Pull request by ansaryergesh**

### CODE REVIEW

**Comment by Oghenebrume50** [comment]
Hi @ansaryergesh , great start to this project everything works fine the code is well organized. However we need to make some few changes, The **More stories** section seems to be in isolated, it should have articles besides it, alter your CSS to make this possible do not add content that was not on the original page, that is, if you have 5 articles in a section, keep it like that. Check HTML validation and fix the errors. Do all that and submit for re-review.

**Comment by bolah2009** [comment]
PR currently been reviewed

File: css/style.css [hunk 1]
```diff
@@ -0,0 +1,615 @@
+*,
+*::after,
+*::before {
+  box-sizing:border-box }
+
+body {
+  max-width: 1350px;
+  margin: 0 auto!important;
+  background-color: rgb(253, 249, 249)!important;
```
Comment (bolah2009) [inline css/style.css:9]: `SUGGESTION`

Always look for a way to use [specificity](https://developer.mozilla.org/en-US/docs/Web/CSS/Specificity) before even considering `!important`. Learn more [here](https://developer.mozilla.org/en-US/docs/Web/CSS/Specificity#The_!important_exception)

File: css/style.css [hunk 2]
```diff
@@ -0,0 +1,615 @@
...
+}
+
+html, body {
+    overflow-x: hidden;
+}
+.container-fluid {
+    position: relative;
```
Comment (bolah2009) [inline css/style.css:16]: `SUGGESTION`

Consider using a Bootstrap class name for this selector element and in all other occurrences.

File: css/style.css [hunk 3]
```diff
@@ -0,0 +1,615 @@
...
+    background-color: white
+}
+
+.heading-none{
+    display: none
```
Comment (bolah2009) [inline css/style.css:21]: `SUGGESTION`

Consider using a Bootstrap class name for this selector element and in all other occurrences.

File: css/style.css [hunk 4]
```diff
@@ -0,0 +1,615 @@
...
+}
+
+.nav-bar {
+	position: relative;
+	margin: auto;
```
Comment (bolah2009) [inline css/style.css:26]: `SUGGESTION`

Consider using a Bootstrap class name for this selector element and in all other occurrences.

File: css/style.css [hunk 5]
```diff
@@ -0,0 +1,615 @@
...
+  margin: 0 auto!important;
+  background-color: rgb(253, 249, 249)!important;
+}
+
+html, body {
+    overflow-x: hidden;
+}
+.container-fluid {
+    position: relative;
+    background-color: white
```
Comment (bolah2009) [inline css/style.css:17]: `SUGGESTION`

Consider using a Bootstrap class name for this selector element and in all other occurrences.

File: css/style.css [hunk 6]
```diff
@@ -0,0 +1,615 @@
...
+	transform: translate(-50%, 0);
+	top: 15px;
+	left: 50%;
+}
+.logoimg > img {
+	width: 100%;
+}
+
+.datenav {
+	position: relative;
```
Comment (bolah2009) [inline css/style.css:53]: `SUGGESTION`

Consider using a Bootstrap class name for this selector element and in all other occurrences.

File: css/style.css [hunk 7]
```diff
@@ -0,0 +1,615 @@
...
+}
+
+.header-bottom::before {
+	width: 66px;
+	position: absolute;
+	top: 0;
+	right: 0;
+	content: "";
+	display: block;
```
Comment (bolah2009) [inline css/style.css:99]: `SUGGESTION`

Consider using a Bootstrap class name for this selector element and in all other occurrences.

File: css/style.css [hunk 8]
```diff
@@ -0,0 +1,615 @@
...
+	background-color: #f72210;
+	border-left: 12px solid #fff;
+	border-top: 13px solid #f72210;
+}
+
+.page-nav {
+	display: flex;
```
Comment (bolah2009) [inline css/style.css:106]: `SUGGESTION`

Consider using a Bootstrap class name for this selector element and in all other occurrences.

File: css/style.css [hunk 9]
```diff
@@ -0,0 +1,615 @@
...
+	line-height: 24px;
+	font-weight: 700;
+	list-style: none;
+	margin: 0px;
+	padding: 0px 54px 0px 0px;
+}
+
+.page-nav li {
+	flex-grow: 1;
+	text-align: center;
```
Comment (bolah2009) [inline css/style.css:117]: `SUGGESTION`

Consider using a Bootstrap class name for this selector element and in all other occurrences.

File: css/style.css [hunk 10]
```diff
@@ -0,0 +1,615 @@
...
+	font-weight: 400;
+	font-family: Arial, Helvetica, sans-serif;
+}
+
+.fa-search {
+	padding-left: 10px;
+	font-weight: 400;
+}
+.hamburger {
+    color: red
```
Comment (bolah2009) [inline css/style.css:147]: `SUGGESTION`

Consider using a Bootstrap class name for this selector element and in all other occurrences.

File: css/style.css [hunk 11]
```diff
@@ -0,0 +1,615 @@
...
+    border:none;
+    color:white;
+    padding: 2px
+}
+
+/* Footer */
+
+.logo-social {
+    display: flex;
+    align-items: center;
```
Comment (bolah2009) [inline css/style.css:474]: `SUGGESTION`

Consider using a Bootstrap class name for this selector element and in all other occurrences.

**Review by bolah2009 — APPROVED** [review]
## Nice Job [Ansar](../../) :clap: :1st_place_medal: 

The [clone](https://raw.githack.com/ansaryergesh/project-using-bootstrap/develop/index.html) looks good to me and responsive. :+1: 

#### Check out the comments below on using [bootstrap](https://getbootstrap.com/) more effectively. Consider checking them out before merging.

Good luck with your next project. :star2: 

Cheers
[Bola](https://bolabuari.com/)

**Pull request merged**